### PR TITLE
[wasm][debugger] Skip wasm frames if just my code is enabled

### PR DESCRIPTION
**Pull request by thaystg**
Trying to avoid stepping out when there is no source available and justmycode is enabled.
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1733523
Trying to fix this: https://stackoverflow.com/questions/74972911/blazor-wasm-debug-stops-with-frame-not-in-module

Callstack without the PR:
image

Callstack with the PR and JustMyCode enabled:
image

Callstack with the PR and JustMyCode disabled:
image

Stepping out without the PR, and you user can step out forever, it has a lot of calls.
image

Stepping out with the PR - will show pause in the JS functions.
image

TODO - Find a way to detect if it's a function from framework (dotnet.js/blazor.webassembly.js/browserLink) and skip them if JustMyCode is enabled.

### CODE REVIEW

File: src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs [hunk 1]
```diff
@@ -198,6 +198,22 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                                     await SendResume(sessionId, token);
                                     return true;
                                 }
+                            case "mono_wasm_fire_debugger_agent_message_with_data":
+                            case "_mono_wasm_fire_debugger_agent_message_with_data":
```
Comment (ilonatommy) [inline src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs:202]: How does `_mono_wasm_fire_debugger_agent_message_with_data` originate and can't we force to have only e.g. `mono_wasm_fire_debugger_agent_message_with_data`?

Comment (ilonatommy) [reply src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs:202]: Never mind, I just noticed all the cases are in this convention.

**Review by ilonatommy — APPROVED** [review]

**Review by radical — APPROVED** [review]

**Pull request merged**